index.html

```diff
@@ -1,180 +1,371 @@
-    <h1 class="title playfair" >Pritikant Dhal</h1>
-    <p class="my-introduction">
-      Hello! I am a software developer! I can help you build a product, 
-      feature or website. Take a look at my work.<br/><br/>
-      If you like what you see and have a project you need coded,
-      don’t hesitate to contact me.
-    </p>
-      <button class="btn">Start collaboration</button>  
-      <figure class="main-end">
-        <img src="images/icons/image_geometry_2.svg" alt="main section background image">
-      </figure>
-  </section>
-  
-  <!--Project Section-->
-  <section id="projects" class="popins">
-  <h3 id="project-title" class="playfair title">projects</h3>
+  </head>
+  <body>
+    <!-- header section -->
+    <header class="flex">
+      <img src="Images/icons/image_geometry_1.svg" alt="" class="l-t-img" />
+      <nav class="nav">
+        <ul class="nav_links popins">
+          <li><a href="" class="nav_link">Hello</a></li>
+          <li><a href="" class="nav_link">Portfolio</a></li>
+          <li><a href="" class="nav_link">About</a></li>
+          <li><a href="" class="nav_link">Contact</a></li>
+        </ul>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke-width="1.5"
+          stroke="currentColor"
+          class="w-6 h-6"
+          id="menu-1"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+          />
+        </svg>
+      </nav>
+    </header>
 
-<article class="project flex">
-  <img src="images/icons/Project-1.svg" alt="Project1 Image1" class="project-img">
-    <h3>Project name goes here</h3>
-   <ul class="project-languages flex">
-      <li class="project-language">HTML/CSS</li>
-      <li class="project-language">Ruby on Rails</li>
-      <li class="project-language">JavaScript</li>
-    </ul>
-  <button class="project-btn flex">
-    <span>See this project</span>
-    <img
-       src="images/icons/ic_arrow_right.svg" width="20px" height="15px" alt="#" srcset="">
-  </button>
-</article>
+    <!-- hero content section -->
+    <sections id="main" class="flex popins">
+      <div class="main-container flex">
+        <h1 class="title playfair">Pritikant Dhal</h1>
+        <p class="my-intro">
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Take a look at my work. <br /><br />
+          If you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <button class="btn popins" id="start-collab-btn">
+          start collabration
+        </button>
+      </div>
+      <div class="main-start"></div>
+      <div class="main-end">
+        <img
+          src="Images/icons/image_geometry_2.svg"
+          alt="main section background"
+        />
+      </div>
+    </sections>
```

Comment:
All images are showing in local machine 
How can I do this.

Reply:
Change capital `i` with small `i` of images like `Images` with `images` then it will work because you folder name is `images` not `Images` so kindly fix it like that 😊